client/src/components/WorkflowInvocationState/WorkflowInvocationBiocompute.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div v-if="Object.keys(invocation).length">
+        <details class="invocation-biocompute-objects">
```

Comment:
Just a minor point, but if you find yourself doing a lot of these kinds of calculations in-template, it might be easier and read better to use an appropriately named computed. 

```js
export default {

    // ....

    computed: {
        hasInvocationProps() {
            return Object.keys(this.invocation).length > 0;
        }
    },

    // ....

}
```